content/docs/next-release/operator.md

```diff
@@ -0,0 +1,645 @@
+---
+title: Operators
+hasparent: true
+weight:
+---
+
+# Understanding Operators
+
+The Jaeger Operator is an implementation of a [Kubernetes Operator](https://coreos.com/operators/). It provides a method of packaging, deploying, and managing Jaeger.
+
+Operators are pieces of software that ease the operational complexity of running another piece of software. More technically, _Operators_ are a method of packaging, deploying, and managing a Kubernetes application.
+
+A Kubernetes application is an app that is both deployed on Kubernetes and managed using the Kubernetes APIs and `kubectl` (kubernetes) or `oc` (OKD) tooling. To be able to make the most of Kubernetes, you need a set of cohesive APIs to extend in order to service and manage your apps that run on Kubernetes. Think of Operators as the runtime that manages this type of app on Kubernetes.
+
+
+# Installing the Operator
+
+IMPORTANT: The Jaeger Operator version is related to the version of the Jaeger components (Query, Collector, Agent) up to the minor portion. The patch version portion does *not* follow the ones from the Jaeger components. For instance, the Operator version 1.8.1 uses the Jaeger Docker images tagged with version 1.8 by default.
+
+NOTE: The following instructions will deploy a version of the operator that is using the latest `master` version. If you want to install a particular stable version of the operator, you will need to edit the `operator.yaml` file and specify the version as the tag in the container image and then use the approproiate `apiVersion` for the Jaeger operator.
+
+
+
+|Up to version |apiVersion |CRD yaml
+| --- | --- | --- |
+|master |jaegertracing.io/v1 |https://github.com/jaegertracing/jaeger-operator/blob/master/deploy/crds/jaegertracing_v1_jaeger_crd.yaml[jaegertracing_v1_jaeger_crd.yaml]|
```

Comment:
The link has to be fixed in Markdown syntax (it's AsciiDoc)

Reply:
Drat!  I thought I'd fixed all the links.  Good catch.